### ISSUE TITLE
`intercept_request` not defined in Ferrum 0.4

### ISSUE THREAD

**Issue by zkhan** [machinio/cuprite#99]
Using `url_whitelist` and calls to `visit` would result in an `NameError`

```
 NameError:
            undefined local variable or method `intercept_request' for #<Ferrum::Page:0x000056434d93ab48>
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/forwardable.rb:224:in `page'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/forwardable.rb:224:in `goto'
          # ./spec/features/embedding/login_page_spec.rb:508:in `visit_page'
          # ./spec/features/embedding/login_page_spec.rb:47:in `block (6 levels) in <top (required)>'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/bin/rspec:23:in `load'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/bin/rspec:23:in `<top (required)>'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/cli/exec.rb:74:in `load'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/cli/exec.rb:74:in `kernel_load'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/cli/exec.rb:28:in `run'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/cli.rb:463:in `exec'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/cli.rb:27:in `dispatch'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/cli.rb:18:in `start'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/lib/ruby/2.6.0/bundler/friendly_errors.rb:124:in `with_friendly_errors'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/bin/bundle:23:in `load'
          # /home/zkhan/.asdf/installs/ruby/2.6.2/bin/bundle:23:in `<main>'
```

Locking to Ferrum 0.3 resolved the issue for me.

**Comment by route** [comment]
Do you use cuprite from master?

**Comment by zkhan** [comment]
Nope? Just from rubygems.org

**Comment by zkhan** [comment]
OK, if it's been addressed I'll try again without locking, perhaps it was cuprite that needed an bundle update.

**Comment by route** [comment]
@zkhan yep that should be it! Try to update both. Thanks

**Comment by zkhan** [comment]
Double checked and 0.7.0 cuprite does not work with as described ferrum 0.4

**Comment by route** [comment]
Thanks I'll take a look!

**Comment by route** [comment]
@zkhan you are right I didn't release cuprite for the last changes made in ferrum. Doing it now, will be fixed in 0.7.1
